src/main/resources/dev/definitions/prisons/orphanage/ors-daily-discharge-list.json

```diff
@@ -407,6 +407,10 @@
       "version": "1.0.0",
       "render": "HTML",
       "dataset": "3377299/DP3",
+      "filter": {
```

Comment:
This DPD seems kind of similar with ors-scheduled-court-and-transfer-movements. I am not sure if this filtering will fix the duplicates issue.
The query populates the NFR column in a wrong way and we might need to do something like [this](https://dsdmoj.atlassian.net/wiki/spaces/DPR/pages/5190352968/Common+Issues+Identified+In+DPDs).